### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by mordamax**
To help avoiding [such PRs stuck](https://github.com/substrate-developer-hub/substrate-node-template/pull/405) adding automatic reviewers for PRs

### CODE REVIEW

**Comment by NukeManDan** [comment]
This template is not maintained. Upstream is, and ci generates a pr to update onnpolkadot releases.

 No other prs should be made to this repo.

**Comment by sacha-l** [comment]
Happy to be a code owner to help here. I think @lsgunnlsgunn should be one too.

**Comment by NukeManDan** [comment]
@lisa-parity you mean ;)

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @sacha-l @lsgunnlsgunn @al3mart
```
Comment (NukeManDan) [inline CODEOWNERS:1]:
```suggestion
* @sacha-l @lisa-parity 
```

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+* @sacha-l @lisa-parity
```
Comment (NukeManDan) [inline CODEOWNERS:1]:
```suggestion
* @sacha-l @lisa-parity
```

**Review by NukeManDan — APPROVED** [review]

**Pull request merged**